illumination-view/src/main/resources/static/css/bootstrap.css

```diff
@@ -1608,6 +1608,18 @@ pre code {
   margin-right: auto;
   margin-left: auto;
 }
+.map-container-fluid {
+  position: relative;
+  width: 80%;
```

Comment:
nie edytuje się bilbiotek css tylko towrzy się swoje włąsne main.css żeby nie robić problemów 
bootstrap.css to bibloioteka css